OneCommunityGlobal/HighestGoodNetworkApp
# Lawrence - Fix back to top covering numbers on User Management page

# Description
![Screenshot (2898)](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/46981223/f30bcefa-1660-4c01-9703-e13611169035)

## Main changes explained:
- Update file UserManagement.jsx to add extra margins below the pagination menu so there's room for the "Back to Top" button

## How to test:
1. check into current branch
2. do `npm run start:local` to run this PR locally
3. log as admin user
4. go to dashboard→ other links → user management
5. verify that you can see now see the clickable numbers at the bottom of the screen

## Screenshots or videos of changes:
Before:
![Capture 1](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/46981223/2530ec85-7bd4-4c01-9903-6e21e0818317)

After:
![Capture 8](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/46981223/7fd2e87c-cadb-42eb-bbae-4509ac776794)

## Note:
My screen size is smaller than average so I can't view the whole button and page numbers, but it'll probably be visible for your screen.
